### ISSUE TITLE
Style/SymbolProc with stabby lambda

### ISSUE THREAD

**Issue by noproblemwiththat** [rubocop/rubocop#14967]
I have a Rails project using plain RuboCop (no extensions for now).  In one of the classes
```ruby
normalizes :regex, with: ->(regex) { regex.strip }
```
gives me the offense
```
Style/SymbolProc: Pass &:strip as an argument to lambda instead of a block.
```
and corrects to 
```ruby
normalizes :regex, with: lambda(&:strip)
```
presumably because
```ruby
# syntax error
normalizes :regex, with: ->(&:strip)
```
is a syntax error.

Now I like symbol procs and so have the Style/SymbolProc cop enabled, but I also prefer the stabby lambda to the explicit.  I can't see any option which is like "unless it changes the lambda type from stabby to explicit", could I suggest that one be added? (With a less cumbersome name.)

**Comment by lovro-bikic** [comment]
> I have a Rails project using plain RuboCop (no extensions for now)

[`Style/SymbolProc` has an exception for Rails projects](https://docs.rubocop.org/rubocop/latest/cops_style.html#allcops_activesupportextensionsenabled_-false-_default_-stylesymbolproc) for your case, but you need to set `AllCops.ActiveSupportExtensionsEnabled` config to true in your `.rubocop.yml` first:
```yml
AllCops:
  ActiveSupportExtensionsEnabled: true
```
or install a plugin like `rubocop-rails` [which turns the config on by default.](https://github.com/rubocop/rubocop-rails/blob/af61bd025aa664a1efb6f643fa4c816106a92360/config/default.yml#L17-L19)

See also https://github.com/rubocop/rubocop/issues/1485

**Comment by noproblemwiththat** [comment]
Oh dozy me, I'd assumed that required extra gems so didn't try it out -- that fixes my issue so closing, many thanks!
